Sources/Algorithms/Unique.swift

```diff
@@ -42,14 +42,14 @@ extension Sequence {
   /// by the given projection), in the order of the first occurrence of each
   /// unique element.
   ///
-  ///     let animals = ["dog", "pig", "cat", "ox", "dog","cat"]
-  ///     let uniqued = animals(on: { $0.first })
+  ///     let animals = ["dog", "pig", "cat", "ox", "dog", "cat"]
+  ///     let uniqued = animals.uniqued(on: {$0.first})
   ///     print(uniqued)
-  ///     // Prints "["dog", "pig","cat", "ox"]"
+  ///     // Prints '["dog", "pig", "cat", "ox"]'
   ///
   /// - Parameter projection: A projecting closure. `projection` accepts an
   ///   element of this sequence as its parameter and returns a projected
-  ///   value of the same  type.
+  ///   value of the same type having constraint `Hashable`.
```

Comment:
This is not correct: the projected value does *not* need to be of the same type.

But moreover, even when corrected, this statement is still just describing the type of the parameter in words, which is not necessary and doesn't add anything. The challenge here is to teach the user why and how to use a projection function to determine uniqueness.